### PR TITLE
Documentation for adding dashboards

### PR DESCRIPTION
**Pull request by michelle192837**
Adding more documentation for adding dashboards (as well as dashboard groups, dashboard tabs, test groups, and some configuration options) to README.md.

### CODE REVIEW

File: testgrid/config/README.md [hunk 1]
```diff
@@ -7,40 +7,114 @@ build :
...
+```
+test_groups:
+- name: ci-kubernetes-node-kubelet-benchmark
+  gcs_prefix: kubernetes-jenkins/logs/ci-kubernetes-node-kubelet-benchmark
+  test_name_config:
+    name_elements:
+    - target_config: Tests name
+    - target_config: Context
+    name_format: '%s [%s]'
+```
```
Comment (krzyzacy) [inline testgrid/config/README.md]: eh, why's this config related to compare commits?

Comment (michelle192837) [inline testgrid/config/README.md]: Whoops, wrong example. Fixed?

**Comment by krzyzacy** [comment]
nice :-) this looks a lot more helpful!
/lgtm

**Pull request merged**